src/licensedcode/data/licenses/inria-linking-exception.LICENSE

```diff
@@ -6,9 +6,13 @@ category: Copyleft Limited
 owner: INRIA
 homepage_url: http://para.inria.fr/~maranget/hevea/distri/LICENSE
 is_exception: yes
-spdx_license_key: LicenseRef-scancode-inria-linking-exception
+spdx_license_key: QPL-1.0-INRIA-2004-exception
+other_spdx_license_keys:
```

Comment:
The original uses the English spelling of licence and SPDX updated this incorrectly IMHO, terefore:

Add these as rules:
```
As a special exception to the Q Public License, you may develop
application programs, reusable components and other software items
that link with the original or modified versions of the
Generator
and are not made available to the general public, without any of the
additional requirements listed in clause 6c of the Q Public license.
```
because the SPDX tagged text has a such a variant

and
```
As a special exception to the Q Public License, you may develop
application programs, reusable components and other software items
that link with the original or modified versions of the Software
and are not made available to the general public, without any of the
additional requirements listed in clause 6c of the Q Public license.
```
with the license-vs-licence spelling 

and
```
As a special exception to the Q Public Licence, you may develop
application programs, reusable components and other software items
that link with the original or modified versions of the
Generator
and are not made available to the general public, without any of the
additional requirements listed in clause 6c of the Q Public licence.
```